index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="icon" type="image/png" href="./img/fav.png">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Space Tech Expo</title>
+</head>
+<body>
+    <nav class="navbar navbar-dark bg-dark top-nav justify-content-between p-1">
+ <ul class="d-flex justify-content-end top-nav-ul">
+ <li class="top-nav-list">Logout</li>
+ <li class="top-nav-list">My Page</li>
+ <li class="top-nav-list">English</li>
+ <li class="top-nav-list"><a href="#" class="fa fa-twitter fa-lg"></a></li>
+    <li class="top-nav-list"><a href="#" class="fa fa-facebook fa-lg"></a></li>
+</ul>
+    </nav>
+    <nav class="navbar navbar-expand-sm navbar-light bg-light py-3 main-nav-menu">
+        <div class="nav-bar">
+          <a class="navbar-brand" href="./index.html"><img class="logo" src="./img/logo.png" alt="Space logo"></a>
+          <ul class="navbar-nav ms-auto align-items-center nav-items">
+            <li class="nav-item">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">News</a>
+            </li>
+            <li class="nav-item emphasis">
+              <a class="nav-link orange" href="#">Campaign</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    
+    
+    <section class="home-section">
+<img id="toggle_menu" class="hamburger" src="./img/menu.png" alt="Hamburger Menu">   
+<div id="menu" class="main-nav">
+    
+        <ul id="menu-ul" class="d-flex main-nav-ul">
+        <li id="close_menu" class="close_menu">x</li>
+        <li class="top-nav-list"><a href="/about.html">About</a></li>
+        <li class="top-nav-list currunt-tab"><a href="/index.html">Program</a></li>
+        <li class="top-nav-list">Join</li>
+        <li class="top-nav-list">Sponsor</li>
+        <li class="top-nav-list">News</li>
+       </ul>
+    </div>
+<div class="home-contanier">
+           <h1 class="home-title">Space Tech Expo 2021</h1>
+
+           <h2 class="home-haeder">SPACE TECH EXPO EUROPE RETURNS FOR 2021</h2>
+
```

Comment:
- [ ] You have text flowing into other text, kindly fix it.

<img src="https://user-images.githubusercontent.com/33323077/133304922-83937fc2-7527-4684-b6c7-f90c2db1ece7.png" height="250px" />